packages/config/src/index.ts

```diff
@@ -4,21 +4,21 @@
 
 import { ethers } from 'ethers'
 
-export const ATTESTTING_FEE = ethers.utils.parseEther('0.1')
+export const ATTESTTING_FEE = ethers.utils.parseEther('0')
```

Comment:
Why you change default attesting fee to zero?

Reply:
Because in alpha test we pay for the attestingFee and it costs too much and not necessary now.
and it should be worked with `attestingFee=0`

Reply:
Because in alpha test we pay for the attestingFee and it costs too much and not necessary now.
and it should be worked with `attestingFee=0`